tests/e2e/utils/utils.js

```diff
@@ -295,6 +295,34 @@ function extractWorkbenchData(data) {
   return workbenchData;
 }
 
+function getGrayLogSnapshotUrl(targetUrl, since_secs = 30) {
+  let snapshotUrl = null;
+
+  // WARNING: This mappings might change
+  const table = {
+    "staging.osparc.io": "https://monitoring.staging.osparc.io/graylog/",
+    "osparc.io": "https://monitoring.osparc.io/graylog/",
+    "osparc-master.speag.com": "https://monitoring.osparc-master.speag.com/graylog/",
+    "osparc-staging.speag.com": "https://monitoring.osparc.speag.com/graylog/",
+    "osparc.speag.com": "https://monitoring.osparc.speag.com/graylog/",
+  };
```

Comment:
not sure. but why not pass this mapping through ENV variables ? these are available in the CI.

Reply:
Yes, I thought that as well, but I leave that to @Surfict as exercise  ;-)